### DISCUSSION TITLE
Wino Mail 1.9.2: Error when i send mail with attachment: Syntax error - line too long

### DISCUSSION THREAD

**Discussion by Robert0711** [bkaankose/Wino-Mail#388] [Q&A]
Hello,
when i try to send an mail with attachment, it fails and the error message is: Syntax error - line too long.

In the previous versions of wino mail (1.9.0) it worked, in the newest (1.9.2) it fails. What can i do?

**Comment by RoderickMcConnell** [comment]
I have the same problem when I try to forward emails with image content.
